js/environment/index.js

```diff
@@ -54,8 +52,7 @@ module.exports = {
   move: function(dt) {
```

Comment:
wondering if a more descriptive name for this method might be `updateParticlePositions` or something of that sort